tests/test_step_sim.py

```diff
@@ -1,10 +1,9 @@
 import argparse
 import os
-from gymfc.envs.fc_env import FlightControlEnv
```

Comment:
You still need to check out this file, if it was identical to master then it wouldn't show up here. If you look at your commit history on this file https://github.com/MichaelManz/gymfc/commits/demo_docker/tests/test_step_sim.py you'll see what I mean. You need to revert to the original commit, https://github.com/MichaelManz/gymfc/commit/1646b7b87fd24fa4577f330a332f6795f0632b9e#diff-fe3ca8f6b76238cebcc99d0c1c4be067